scala/README.md

```diff
@@ -644,14 +669,17 @@ Tips & Tricks
 
 0. You can use the `@scala.beans.BeanProperty` and `@BooleanBeanProperty` annotations to automatically generate JavaBeans style getter and setter methods.
 
-0. Use this syntax to pass a sequence as a parameter to a variable length argument list method:
+0. If a method takes a parameter of type `Seq[A]`, put that parameter last and make it a variable-length argument (_vararg_). Use the syntax below to pass a sequence as a parameter to the method, if needed:
```

Comment:
@matthandlersux This is under the "Tips & Tricks" section, which is totally optional. I've reworded it to make it clear it is not required.